lyricsgenius/api.py

```diff
@@ -276,6 +427,105 @@ def search_artist(self, artist_name, verbose=True, max_songs=None, take_first_re
 
         return artist
 
+    def search_album(self, artist_name, album_title):
+        """Get all lyrics from an album"""
+        # authentify
+        api = Genius(client_access_token)
```

Comment:
The user already does this when they import the library. Instead of creating a new instance of the Genius class, you should remove line 433 and replace `api` with `self` throughout this function.